angr/analyses/decompiler/structured_codegen.py

```diff
@@ -109,6 +93,18 @@ def __init__(self):
         pass
 
     def c_repr(self, indent=0, posmap=None):
+
+        def mapper(chunks, posmap):
+            pos = 0
+            for s, obj in chunks:
+                if obj:
```

Comment:
Are we going to miss the case where `obj == 0`? I prefer a more explicit check here, aka `if obj is not None:`.